# This contains a list of Odometry type messages
# It is used to keep positions and velocities of every agent in the system
std_msgs/Header header
nav_msgs/Odometry[] array